src/routes/transactions/__tests__/controllers/list-multisig-transactions-by-safe.transactions.controller.spec.ts

```diff
@@ -168,18 +164,14 @@ describe('List multisig transactions by Safe - Transactions Controller (Unit)',
       status: 200,
     });
     networkService.get.mockResolvedValueOnce({
-      data: { ...page, count: null },
+      data: { ...page, count: '1' },
```

Comment:
`count` is nullable according to our types, and the Transaction Service.

Reply:
If `count` is nullable why did we have to change the `count` from `null` to `1` here? 🤔